tests/translator/test_translator.py

```diff
@@ -138,7 +138,122 @@ def mock_sar_service_call(self, service_call_function, logical_id, *args):
 # api and s3 location for explicit api.
 
 
-class TestTranslatorEndToEnd(TestCase):
+class TestTranslator(TestCase):
```

Comment:
nit: would rename the test class to `AbstractTestTranslator` since it doesn't contain actual test in it.